linera-storage/src/db_storage.rs

```diff
@@ -742,15 +743,15 @@ where
         Ok(())
     }
 
-    fn add_blob_to_batch(
+    fn add_blob_content_to_batch(
```

Comment:
I'd actually call this `add_blob_to_batch` and give it a single `Blob` argument instead of the separate content and ID: That way, the caller is less likely to make a mistake and combine the wrong ID with a content.